default.yaml

```diff
@@ -125,6 +125,18 @@ rateLimits:
 
 accounts:
   initialStatus: active
+  # customFields:
```

Comment:
I'm thinking we may need a `unique` property also?

(it almost like we're redefining a schema language here. wondering if there could be an approach where we just use a library? this would remove some burden on the maintenance)

Reply:
maybe this is what `index` does. not sure to me.

Reply:
we create an index but not a unique index. Also this is a bit tricky to implement because need to have track of changes (duplicated records).

let me know if you want to include it in this PR

Reply:
`unique` feels more important than `array` or `default`.

Reply:
Unique added here https://github.com/GaloyMoney/galoy/pull/1465/commits/365d353f84a0c792aac7b84ddf202e2cc7c96f44